github/enterprise_codesecurity_configurations.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// GetCodeSecurityConfigurations lists all code security configurations available in an enterprise.
+//
+// GitHub API docs: https://docs.github.com/rest/code-security/configurations#get-code-security-configurations-for-an-enterprise
+//
+//meta:operation GET /enterprises/{enterprise}/code-security/configurations
+func (s *EnterpriseService) GetCodeSecurityConfigurations(ctx context.Context, enterprise string, opts *GetCodeSecurityConfigurationOptions) ([]*CodeSecurityConfiguration, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/code-security/configurations", enterprise)
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var configurations []*CodeSecurityConfiguration
+	resp, err := s.client.Do(ctx, req, &configurations)
+	if err != nil {
+		return nil, resp, err
+	}
+	return configurations, resp, nil
+}
+
+// CreateCodeSecurityConfiguration creates a code security configuration in an enterprise.
+//
+// GitHub API docs: https://docs.github.com/rest/code-security/configurations#create-a-code-security-configuration-for-an-enterprise
+//
+//meta:operation POST /enterprises/{enterprise}/code-security/configurations
+func (s *EnterpriseService) CreateCodeSecurityConfiguration(ctx context.Context, enterprise string, config *CodeSecurityConfiguration) (*CodeSecurityConfiguration, *Response, error) {
```

Comment:
Mandatory settings we pass by value.


```suggestion
func (s *EnterpriseService) CreateCodeSecurityConfiguration(ctx context.Context, enterprise string, config CodeSecurityConfiguration) (*CodeSecurityConfiguration, *Response, error) {
```

Reply:
sorry, I didn't know (and it wasn't the case for the org endpoints)